common/build.gradle.kts

```diff
@@ -56,7 +56,7 @@ fun getGitCommitId(): String {
 }
 
 val propertiesFile = "src/main/resources/project.properties"
-val writeProjectPropertiesFile = tasks.register("writeProjectPropertiesFile") {
+fun writeProjectPropertiesFile() {
```

Comment:
Change to function and add to jar's doFirst to make sure it will run always and cannot skip.